File: Week 6/cash.py
from cs50 import get_float

while True:
    dollars = get_float("Change owed: ")
    if dollars > 0:
        break

cents = dollars * 100
coins = 0

coins += int(cents / 25)
coins += (cents % 25) / 10
coins += ((cents % 25) % 10) / 5
coins += ((cents % 25) % 10) % 5

print(int(coins))

File: cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    float dollars;
    do
    {
        dollars = get_float("Change owed: ");
    }
    while (dollars < 0);
    int cents = round(dollars * 100);  // money conversion
    int coins = 0;
    coins += cents / 25;    
    coins += (cents % 25) / 10;
    coins += ((cents % 25) % 10) / 5;
    coins += ((cents % 25) % 10) % 5;
    printf("%i\n", coins);
}
